snuba/web/query.py

```diff
@@ -51,7 +60,9 @@ def raw_query(
     request: Request,
     query: DictClickhouseQuery,
     timer: Timer,
+    query_metadata: SnubaQueryMetadata,
     stats: Optional[MutableMapping[str, Any]] = None,
+    trace_id: str = "",
```

Comment:
nit: if we are going to pass this trace id around a lot, I would advise to add a type alias, it makes it harder to use it in the wrong place (that expect a string, but conceptually does not expect a trace id).